geode-core/src/main/java/org/apache/geode/distributed/AbstractLauncher.java

```diff
@@ -668,13 +747,45 @@ public String getJavaVersion() {
      * 
      * @return a List of String value each representing an argument passed to the JVM of the GemFire
      *         service.
+     *
      * @see java.lang.management.RuntimeMXBean#getInputArguments()
      */
     public List<String> getJvmArguments() {
       return jvmArguments;
     }
 
     /**
+     * Gets the arguments passed to the JVM process that is running the GemFire service.
+     * 
+     * @return a List of String value each representing an argument passed to the JVM of the GemFire
+     *         service.
+     *
+     * @see java.lang.management.RuntimeMXBean#getInputArguments()
+     */
+    public List<String> getRedactedJvmArguments() {
```

Comment:
This method does essentially the same thing as `Gfsh#getRedactedCmdlet`.  I think extracting this logic into a method of a small LogRedactor class which you can call from both places might clean things up a bit.